version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.1.1-SNAPSHOT"
+version in ThisBuild := "2.0.0-SNAPSHOT"
```

Comment:
Please make `1.2.0-SNAPSHOT`, reason being that we would like to keep all Kamon 1.x modules also within the 1.x.x range. This will make it all less confusing when we release Kamon 2.0!

Reply:
sure